.github/workflows/tests.yml

```diff
@@ -445,6 +445,168 @@ jobs:
           name: lxd-clients-${{ runner.os }}
           path: bin/
 
+  ui-e2e-tests:
+    name: UI e2e tests
+    runs-on: ubuntu-latest
+    needs: [code-tests, documentation]
+    if: ${{ github.repository == 'canonical/lxd' && github.event_name == 'push' }}
+    env:
+      LXD_DIR: "/var/lib/lxd"
+      LXD_OIDC_CLIENT_ID: ${{ secrets.LXD_UI_OIDC_CLIENT_ID }}
+      LXD_OIDC_ISSUER: ${{ secrets.LXD_UI_OIDC_ISSUER }}
+      LXD_OIDC_AUDIENCE: ${{ secrets.LXD_UI_OIDC_AUDIENCE }}
+      LXD_OIDC_USER: ${{ secrets.LXD_UI_OIDC_TEST_USER }}
+      LXD_OIDC_PASSWORD: ${{ secrets.LXD_UI_OIDC_TEST_PASSWORD }}
+      LXD_OIDC_GROUPS_CLAIM: "lxd-idp-groups"
```

Comment:
This is missing `LD_LIBRARY_PATH: "/home/runner/go/bin/dqlite/libs/"` which is probably why you have that error about missing `libdqlite.so.0`.

Reply:
We don't need it here, this is the ui test runner in js, it doesn't need to talk to dqlite directly. Only talks to the ui, that is using the LXD api.

Reply:
you need the dqlite libs and that env var to run lxd binary though

Reply:
Yes, that is in another previous step, which is launching the LXD api.